Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -1747,12 +1747,6 @@ function createAnimatedComponent(Component: any): any {
       var callback = () => {
         if (this._component.setNativeProps) {
           if (!this._propsAnimated.__isNative) {
-            if (this._component.viewConfig == null) {
-              var ctor = this._component.constructor;
-              var componentName = ctor.displayName || ctor.name || '<Unknown Component>';
-              throw new Error(componentName + ' "viewConfig" is not defined.');
-            }
-
             this._component.setNativeProps(
```

Comment:
We should ensure viewConfig exists before call setNativeProps

```
if (this._component.viewConfig != null) {
  this._component.setNativeProps(...);
}
```

See https://github.com/facebook/react-native/blob/master/Libraries/Renderer/src/renderers/native/NativeMethodsMixin.js#L141-L156